common-service-core/src/main/java/org/zowe/apiml/config/ApiInfo.java

```diff
@@ -26,6 +27,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
+@SuperBuilder
```

Comment:
I see this for the first time - https://projectlombok.org/features/experimental/SuperBuilder - Bob now knows his ancestors: Builders with fields from superclasses, too.